plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestBaseJdbcConfig.java

```diff
@@ -63,14 +64,17 @@ public void testExplicitPropertyMappings()
     }
 
     @Test
-    public void testConnectionUrlIsValid()
+    public void testConnectionUrlIsValidated()
     {
         assertThatThrownBy(() -> buildConfig(ImmutableMap.of("connection-url", "jdbc:")))
                 .hasMessageContaining("must match the following regular expression: ^jdbc:[a-z0-9]+:(?s:.*)$");
         assertThatThrownBy(() -> buildConfig(ImmutableMap.of("connection-url", "jdbc:protocol")))
                 .hasMessageContaining("must match the following regular expression: ^jdbc:[a-z0-9]+:(?s:.*)$");
-        buildConfig(ImmutableMap.of("connection-url", "jdbc:protocol:uri"));
-        buildConfig(ImmutableMap.of("connection-url", "jdbc:protocol:"));
+
+        assertThatCode(() -> buildConfig(ImmutableMap.of("connection-url", "jdbc:protocol:uri")))
+                .doesNotThrowAnyException();
```

Comment:
Why is the difference between previous code? Maybe we should rename `buildConfig` to `assertJdbcConfigIsValid`?

Reply:
I like the change. The earlier `buildConfig` looked like a no-op and it wasn't obvious it throws on failure.

Reply:
Reverted